projects/Mallard/src/authentication/authorizers/CASAuthorizer.ts

```diff
@@ -16,6 +17,7 @@ export default new Authorizer({
         legacyCASUsernameCache,
         legacyCASPasswordCache,
     ] as const,
+    validAttemptCache: validAttemptCache,
```

Comment:
I'm not certain but I think with this new implementation the individual authorisers no longer need their own validAttemptCache?

Reply:
Good point since the cache is only read and written to in the AccessController